vars/perfSearchBestConfigParallelPipeline.groovy

```diff
@@ -174,7 +174,7 @@ def call(Map pipelineParams) {
                                     withEnv(["AWS_ACCESS_KEY_ID=${env.AWS_ACCESS_KEY_ID}",
                                              "AWS_SECRET_ACCESS_KEY=${env.AWS_SECRET_ACCESS_KEY}",
                                              "SCT_TEST_ID=${UUID.randomUUID().toString()}",
-                                             "SCT_GCE_PROJECT=${env.SCT_GCE_PROJECT}",]) {
+                                             "SCT_GCE_PROJECT=${env.SCT_GCE_PROJECT ?: ''}",]) {
```

Comment:
`params.gce_project` ?

Reply:
I don't know, I don't have time to fuss around with this silly thing.

if it's working it's working.